tensorflow/contrib/lite/java/demo/app/build.gradle

```diff
@@ -44,7 +44,7 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestCompile('com.androidx.test.espresso:espresso-core:2.2.2', {
+    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
```

Comment:
Could you try: 
   `￼androidx.test.espresso:espresso-core:3.1.0-alpha3`
￼instead?

Reply:
Done